python/ray/serve/_private/replica.py

```diff
@@ -690,6 +694,14 @@ async def initialize(self, deployment_config: DeploymentConfig):
                     self._user_callable_asgi_app = await asyncio.wrap_future(
                         self._user_callable_wrapper.initialize_callable()
                     )
+                    if self._user_callable_asgi_app:
+                        docs_path = self._user_callable_wrapper._callable.docs_path
+                        # Since it's possible for user to pass a builder function that returns a FastAPI app
+                        # we will not have a docs path on the ASGI app object untill initialize_callable is called
+                        # so we set the docs path on the controller in this case
+                        await self._controller_handle.set_docs_path.remote(
+                            self._deployment_id.app_name, docs_path
+                        )
```

Comment:
A problem here that is not critical but annoying, if there are 100 replicas, then we would be making 100 controller calls

Reply:
that sounds much better. ty